rest-api/src/test/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1Test.java

```diff
@@ -62,6 +63,17 @@ void testGetFileById() throws IOException {
 				.isEqualTo("1234");
 	}
 
+	@Test
+	void testMarkFileAsProcessed() {
+		when(cpcFileService.processFileById(anyString())).thenReturn("success!");
+
+		ResponseEntity<String> response = cpcFileControllerV1.markFileProcessed("meep");
+
+		verify(cpcFileService, times(1)).processFileById(anyString());
```

Comment:
I would verify that it calls `processFileById` with `meep` as the argument instead of `anyString()`.

Reply:
👍